rules/less-is-more/rule.md

```diff
@@ -0,0 +1,61 @@
+---
+type: rule
+title: Do you know that Less is more ?
+uri: less-is-more
+authors:
+  - title: Josep Fernandez
+    url: https://www.ssw.com.au/people/joseph-fernandez
+created: 2023-02-22T04:20:12.183Z
+guid: 9d8ce87b-f835-48a2-b60c-5549466da209
+---
+A popular concept in product development and interphase design is the idea of “Less is more”.  But what do we actually mean by less is more? 
+
```

Comment:
This rule is missing the markdown tag that separates the blurb view from full content:

`<!--endintro-->`

I would add it here

1. Please action